test/pylib/rest_client.py

```diff
@@ -311,6 +311,10 @@ async def backup(self, node_ip: str, ks: str, tag: str, dest: str, bucket: str)
         """Backup keyspace's snapshot"""
         return await self.client.post_json(f"/storage_service/backup?keyspace={ks}&endpoint={dest}&bucket={bucket}&snapshot={tag}", host=node_ip)
 
+    async def take_snapshot(self, node_ip: str, ks: str, tag: str) -> None:
+        """Take keyspace snapshot"""
+        await self.client.post(f"/storage_service/snapshots?kn={ks}&tag={tag}", host=node_ip)
```

Comment:
nit, instead of embedding the parameters directly in the URL, it's recommended to use the `params` parameter in your request. this approach is cleaner, more readable, and helps avoid potential URL encoding issues.